requirements/common.txt

```diff
@@ -19,7 +19,7 @@ tiktoken >= 0.6.0  # Required for DBRX tokenizer
 lm-format-enforcer >= 0.10.11, < 0.11
 outlines == 0.1.11
 lark == 1.2.2
-xgrammar == 0.1.11; platform_machine == "x86_64"
+xgrammar >= 0.1.15; platform_machine == "x86_64" or platform_machine == "aarch64"
```

Comment:
I would prefer to pin considering we've had breaking changes in upgrades before
```suggestion
xgrammar == 0.1.15; platform_machine == "x86_64" or platform_machine == "aarch64"
```

Reply:
OK, fair enough.

Reply:
done!